
# The background color of the objects to be segmented
black_white_background: false    # false for black

# choice for segmentation or blending
segment_dirt_or_blend_: true   # false for segmentation

# Number of blended images to be generated
blended_img_num_: 0

# clean ground images path 
ground_image_path_: /home/rmb-jx/dataset_new/ground_image

# to be segmented dirt images path
dirts_to_segmented_path_: /home/rmb-jx/dataset(objects)/Others

# Path to save the segmented dirt after cropping, used for image blending
segmented_dirt_cropped_path_:  /home/rmb-jx/dataset_new/Dirt  #/home/robot/dataset(objects)/segmented_dirts #

# Path to save the cropped dirt masks
segmented_dirt_cropped_mask_path_: /home/rmb-jx/dataset_new/Dirtmask #/home/robot/dataset(objects)/segmented_dirts_mask #

# Segmented pens path, for blending normal objects
segmented_pens_path_: /home/rmb-jx/dataset_new/Object    #/home/robot/dataset/segmented_stationary #

# Segmented pens mask path
segmented_pens_mask_path_: /home/rmb-jx/dataset_new/Objectmask         #/home/robot/dataset/segmented_stationary_mask #

# Path to save the blended images
blended_ground_image_path_: /home/rmb-jx/dataset/blended_ground_image

# Path to save the blended image masks
blended_ground_image_mask_path_: /home/rmb-jx/dataset/blended_ground_mask

# Border residual for cropping bounding box
crop_residual_: 2

# maximum number of dirts per frame
max_num_dirts_: 8

# minimum number of dirts per frame
min_num_dirts_: 5

# maximum number of dirts per frame
max_num_pens_: 6

# minimum number of dirts per frame
min_num_pens_: 2

# Path to the output file which storages the arguments of the bounding boxes
bbox_argus_path_: /home/rmb-jx/dataset_new/bboxArgus.txt

# Iterative times for the same ground pattern
ground_pattern_iterative_times_: 2

# Option , if to flip the clean ground image or not horizontally
flip_clean_ground_ : 1

#  brightness shadow mask path
brightness_shadow_mask_path_: /home/rmb-jx/dataset/brightness_shadow_mask